tests/src/rules/dynamic-import-chunkname.js

```diff
@@ -32,6 +32,14 @@ ruleTester.run('dynamic-import-chunkname', rule, {
       )`,
       options,
     },
+    {
+      code: `dynamicImport(
+        /* webpackInclude: /some-regex/ */
+        /* webpackChunkName: "someModule" */
```

Comment:
Must the comments be in this order, or can they be swapped?

Either way, please also add a test for the reversed order, and for "webpackExclude"

Reply:
> Must the comments be in this order, or can they be swapped?
They can be swapped.
> Either way, please also add a test for the reversed order, and for "webpackExclude"
OK.